### PR TITLE
Header is added to checkout component

### PR DESCRIPTION
**Pull request by forjoellobo**

### CODE REVIEW

File: src/screens/home/Home.js [hunk 1]
```diff
@@ -104,11 +100,10 @@ class Home extends Component {
     return (
       <div>
         <div>
-          {/* <Header
-            {...this.props}
-            onChange={this.searchChangeHandler}
-            isHomePage={true}
-          /> */}
+          {
+            <Header logoutHandler={this.loginredirect} baseUrl={Constants.API_BASE_URL}
```
Comment (ashwini008) [inline src/screens/home/Home.js:111]: login redirect method definition is missing

File: src/screens/home/Home.js [hunk 2]
```diff
@@ -104,11 +100,10 @@ class Home extends Component {
...
+              searchRestaurantByName={this.searchRestaurantByName.bind(this)} showSearch={true} history={this.props.history} />
```
Comment (ashwini008) [inline src/screens/home/Home.js:112]: searchRestaurantByName() definition missing

File: src/screens/home/Home.js (hunk 1 above)
Comment (forjoellobo) [reply src/screens/home/Home.js:111]: I am fixing it

Comment (forjoellobo) [reply src/screens/home/Home.js:111]: added

File: src/screens/home/Home.js (hunk 2 above)
Comment (forjoellobo) [reply src/screens/home/Home.js:112]: It is there. I just modified the existing code

**Pull request merged**